pkg/splunk/enterprise/configuration.go

```diff
@@ -584,7 +603,15 @@ func updateSplunkPodTemplateWithConfig(client splcommon.ControllerClient, podTem
 		if instanceType == SplunkStandalone {
 			podTemplateSpec.ObjectMeta.Annotations[smartStoreConfigRev] = smartstoreConfigMap.ResourceVersion
 		}
+	}
+
+	appListingConfigMap, needToMountAppListingCm := getAppListingConfigMap(client, cr, instanceType)
```

Comment:
Do we actually need `needToMountAppListingCm` or just checking if `appListingConfigMap!=nil` should do the trick?

Reply:
Both work. getAppListingConfigMap internally returns both the values, and we are using one of that :-)